mocca-jaxrs2/src/main/java/com/paypal/mocca/client/MoccaJaxrsClient.java

```diff
@@ -47,97 +50,115 @@ public Client build() {
 
         @Override
         public ClientBuilder withConfig(Configuration config) {
-            throw new UnsupportedOperationException(USAGE_ERR_MSG);
+            log.debug(USAGE_ERR_MSG);
```

Comment:
What about using warning level instead of debug? I think it would be more appropriate for this case.